docs/optimization_guide/nncf/ptq/code/ptq_torch_fx.py

```diff
@@ -0,0 +1,58 @@
+# Copyright (C) 2018-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+#! [dataset]
+import nncf
+import torch
+
+calibration_loader = torch.utils.data.DataLoader(...)
+
+def transform_fn(data_item):
+    images, _ = data_item
+    return images
+
+calibration_dataset = nncf.Dataset(calibration_loader, transform_fn)
+#! [dataset]
+
+#! [quantization]
+import torchvision
+import openvino.torch
+model = torchvision.models.resnet50(pretrained=True)
+
+input_fp32 = ... # FP32 model input
+
+with nncf.torch.disable_patching():
+    # Using capture_pre_autograd_graph to export torch.fx.GraphModule
+    # the same way it done for PyTorch 2 Export Post Training Quantization:
+    # https://pytorch.org/tutorials/prototype/pt2e_quant_ptq.html
+    captured_graph = torch._export.capture_pre_autograd_graph(model, args=(input_fp32,))
+    quantized_model = nncf.quantize(captured_graph, calibration_dataset)
+#! [quantization]
+
+#! [inference]
+import openvino as ov
+
+input_fp32 = ... # FP32 model input
+
+# compile quantized model using torch.compile API
+with nncf.torch.disable_patching():
+    compiled_model_int8 = torch.compile(quantized_model, backend="openvino")
+    # First call compiles an OpenVino model underneath, so it could take longer
+    # than original model call.
+    res = compiled_model_int8(input_fp32)
+...
+
+
+# convert exported Torch model to OpenVINO model
+with nncf.torch.disable_patching():
+    exported_quantized_model = torch.export.export(quantized_model, args=(input_fp32,))
+    ov_quantized_model = ov.convert_model(quantized_model, example_input=input_fp32)
+
+# compile the model to transform quantized operations to int8
+model_int8 = ov.compile_model(ov_quantized_model)
```

Comment:
```suggestion
model_int8 = ov.compile_model(ov_quantized_model, 'CPU')
```

Reply:
Should we do the same in https://github.com/openvinotoolkit/openvino/blob/master/docs/optimization_guide/nncf/ptq/code/ptq_torch.py, https://github.com/openvinotoolkit/openvino/blob/master/docs/optimization_guide/nncf/ptq/code/ptq_tensorflow.py, https://github.com/openvinotoolkit/openvino/blob/master/docs/optimization_guide/nncf/ptq/code/ptq_openvino.py, https://github.com/openvinotoolkit/openvino/blob/master/docs/optimization_guide/nncf/ptq/code/ptq_onnx.py, https://github.com/openvinotoolkit/openvino/blob/master/docs/optimization_guide/nncf/ptq/code/ptq_aa_openvino.py, https://github.com/openvinotoolkit/openvino/blob/master/docs/optimization_guide/nncf/ptq/code/ptq_aa_onnx.py as well?